pandas/core/groupby/groupby.py

```diff
@@ -581,6 +581,16 @@ def wrapper(*args, **kwargs):
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
+            if name == 'hist' and kwargs.get('equal_bins', False):
+                # GH-22222
+                bins = kwargs.get('bins')
```

Comment:
do not  add anything here

this is obscuring already extremely fragile code

Reply:
Do you mean not in the wrapper? Or also beyond that? Is there any other place you'd suggest?